Run Google App Engine on Fedora (python)

```diff
@@ -0,0 +1,27 @@
+I faced a lot of challenges while making Google App Engine run on my Fedora platform. So, to tackle the exact problems faced by me, I am writing this post:
```

Comment:
Make it clear that you are talking about Python here. And no need to open like that.

Just say that this articles covers what it covers. You can add it after that it might be complicated for a new developer, and give your case as example. In fact, you can even provide examples of what problems you faced at what stage and how you resolved that.

That would make the article all the more helpful and information rich.